pyrasterframes/src/main/python/tests/RasterFunctionsTests.py

```diff
@@ -286,7 +286,7 @@ def test_render_composite(self):
         cat = self.spark.createDataFrame([
             Row(red=self.l8band_uri(4), green=self.l8band_uri(3), blue=self.l8band_uri(2))
         ])
-        rf = self.spark.read.raster(catalog = cat, catalog_col_names=['red', 'green', 'blue'])
+        rf = self.spark.read.raster(catalog=cat, catalog_col_names=cat.columns)
```

Comment:
In other examples you've dropped the named keyword parameter.... are you testing that `catalog` still works here, or should it be dropped?

Reply:
yes for back compatibility it is good to have at least one ; but probably better to do it in a single unit test so the intent is clear.